### PR TITLE
[FLINK-32863][runtime-web] Improve Flink UI's time precision from sec…

### PR DESCRIPTION
**Pull request by hejufang**
## What is the purpose of the change

Improve Flink UI's time precision from second level to millisecond level.

## Brief change log

Improve Flink UI's time precision from second level to millisecond level, includes job duration, task duration, task start time, end time.
img_v2_72cdb68a-d3dc-4b72-b7c5-4d596893754g
## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? yes
  - If yes, how is the feature documented? not documented

### CODE REVIEW

File: flink-runtime-web/web-dashboard/src/app/components/humanize-duration.pipe.ts [hunk 1]
```diff
@@ -46,23 +46,23 @@ export class HumanizeDurationPipe implements PipeTransform {
...
-            return `${hours}h ${minutes}m ${seconds}s`;
+            return `${hours}h ${minutes}m ${seconds}s ${ms}ms`;
           }
         }
       } else {
         if (short) {
           return `${days}d ${hours}h`;
         } else {
-          return `${days}d ${hours}h ${minutes}m ${seconds}s`;
+          return `${days}d ${hours}h ${minutes}m ${seconds}s ${ms}ms`;
```
Comment (KarmaGYZ) [inline flink-runtime-web/web-dashboard/src/app/components/humanize-duration.pipe.ts:65]: nit: I think the explicit ms might be useless for jobs longer than 1h.

Comment (hejufang) [reply flink-runtime-web/web-dashboard/src/app/components/humanize-duration.pipe.ts]: Thanks for your suggestion, it has been fixed.

**Review by masteryhx — COMMENTED** [review]
Thanks for the contribution.
Do we also need to adjust the precision of checkpoint related time ?
I have seen some cases that checkpoint completes within 1s then the Checkpoint UI will show  same value about start time and end time.
cc @KarmaGYZ

**Comment by KarmaGYZ** [comment]
@masteryhx +1 for also adjusting checkpoint precision.

**Comment by hejufang** [comment]
@masteryhx Thank you for your suggestion.  I have adjusted the precision of checkpoint related time.  please review. cc @KarmaGYZ 
img_v2_a30822c5-c560-4789-9226-ca3899483b7g

**Review by KarmaGYZ — APPROVED** [review]
LGTM.

**Pull request merged**